app/src/main/res/layout/list_patientsummary_add_new_bp_measurement.xml

```diff
@@ -14,7 +15,7 @@
     android:paddingStart="@dimen/spacing_24"
     android:paddingTop="@dimen/spacing_24">
 
-    <android.support.v7.widget.AppCompatButton
+    <Button
```

Comment:
This was intentionally done because `app:backgroundTint` is an attribute of `AppCompatButton`.

Reply:
Same in other places.